docs/docs/introduction/getting-started.md

```diff
@@ -18,7 +18,7 @@ If on a VPS/KVM, and there is already an admin account, either delete that defau
 | **CPU**              | 1 core, 64-bit                                 | 4 cores                              |
 | **Memory**           | 1 GB (no SpamAssassin and ClamAV)              | 4 GB                                 |
 | **Disk**             | 10 GB HDD                                      | 40 GB SSD                            |
-| **Operating System** | Debian 10, 11 or 12<br>Ubuntu 20.04, 22.04 LTS | Latest Debian <br> Latest Ubuntu LTS |
+| **Operating System** | Debian 10, 11 or 12<br>Ubuntu 22.04, 24.04 LTS | Latest Debian <br> Latest Ubuntu LTS |
```

Comment:
We can also remove Debian 10

Reply:
Strictly 20.04 is also supported but I prefer users not install them as they go out of support in 2 months..